domains/uvesh.is-a.dev.json

```diff
@@ -0,0 +1,12 @@
+{
+    "description": "uvesh's .is-a.dev domain",
+    "owner": {
+      "username": "UveshRajwani",
+      "email": "uveshrajwani786@gmail.com"
+    },
+    "record": {
+       "A": [
+      "167.86.86.55"
```

Comment:
I just get the default Nginx landing page on port 80. What port is your website on?

Reply:
yes i am making the website for now i have the default page

Reply:
@UveshRajwani You must have a website to get a domain.